hooks/post_gen_project.py

```diff
@@ -492,10 +492,6 @@ def main():
         )
 
     if "{{ cookiecutter.cloud_provider }}" == "None":
-        print(
-            WARNING + "You chose not to use a cloud provider, "
-            "media files won't be served in production." + TERMINATOR
-        )
```

Comment:
I've removed that but actually we should probably leave this warning if not using docker.

Reply:
Thank you @browniebroke. We have been waiting for this to get into the `master` branch. I have maintained my own branch based on @arrys work. Let me see what we can add to it, especially in the documentation.